pkg/rpm/postupgrade

```diff
@@ -139,16 +139,25 @@ docker run --rm -d --name $SVC_NAME -p 9200:9200 \
   sh -c "$ELASTIC_BIN -f -Des.node.name=$NODE_NAME \
   -Des.cluster.name=$CLUSTER_NAME"
 
-if check_elasticsearch; then
-    report "SUCCESS" "Container started within timeout"
+#
+# CC-4419: If Elasticsearch configuration file and data folder does not exist
+# on the delegate that migration shouldn't start
+#
+if [[ ! -d $HOST_ES_DATA_DIR ] || [ ! $CLUSTER_NAME ]]; then
```

Comment:
If this is supposed to run in a bash shell, this syntax won't work.  It should be
```
if [[ ! -d $HOST_ES_DATA_DIR || ! $CLUSTER_NAME ]]; then
```

Reply:
Example:
```
$ if [[ ! -d /blah ] || [ ! "" ]]; then echo "hi"; fi
-bash: syntax error in conditional expression
-bash: syntax error near `]'
$ if [[ ! -d /blah || ! "" ]]; then echo "hi"; fi
hi
```

Reply:
yeah. my fail